### PR TITLE
Test cases for update, disable and delete user endpoints

### PR DESCRIPTION
**Pull request by wobo-apisero-shikhar**
Implemented test cases for the following User SCIM APIs:
- update user
- disable user
- delete user

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (wobo-apisero-vidhya) [inline cypress/fixtures/example.json]: This file is not required

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (wobo-apisero-vidhya) [inline cypress/support/commands.js]: pls remove this file

File: package.json [hunk 3]
```diff
@@ -8,7 +8,10 @@
     "cy:open": "cypress open"
   },
   "devDependencies": {
-    "@bahmutov/cy-api": "1.5.0",
-    "cypress": "7.2.0"
+    "@bahmutov/cy-api": "^1.6.2",
+    "cypress": "^9.3.1"
+  },
```
Comment (wobo-apisero-vidhya) [inline package.json]: Lets avoid the package imports random-email here and use the fixture test account. We can create , update, and delete the dummy user account.

Comment (wobo-apisero-shikhar) [reply package.json]: done

File: cypress/support/commands.js (hunk 2 above)
Comment (wobo-apisero-shikhar) [reply cypress/support/commands.js]: done

**Review by wobo-apisero-vidhya — APPROVED** [review]

**Pull request merged**